apps/sensenet/src/components/field-controls/webhook-trigger.tsx

```diff
@@ -132,7 +132,8 @@ export const WebhookTrigger: React.FC<ReactClientFieldSetting<LongTextFieldSetti
   const logger = useLogger('webhook filter')
   const localization = useLocalization()
 
-  const initialState = {
+  const initialState = (props.settings.DefaultValue &&
```

Comment:
we should only use the DefaultValue on new view

Reply:
props.fieldValue cannot be empty, only in new mode I think. but okay 👍